### PR TITLE
miau miau miau miau 

### PR DESCRIPTION
**Pull request by DanyPalma**
let me cook with the update hacker form
* prepopulate data fields
* actually let the form submit
image

### CODE REVIEW

File: apps/admin/src/app/(dashboard)/hackers/update-hacker-form.tsx [hunk 1]
```diff
@@ -29,28 +29,43 @@ export function UpdateHackerForm({
...
+        | undefined,
     },
   });
 
   return (
     <Form {...form}>
       <form
         onSubmit={form.handleSubmit((data) => {
-          createHacker.mutate({
+          console.log("Form Data:", data);
```
Comment (Lewin-B) [inline apps/admin/src/app/(dashboard)/hackers/update-hacker-form.tsx]: get rid of these

File: apps/admin/src/app/(dashboard)/hackers/update-hacker-form.tsx [hunk 2]
```diff
@@ -29,28 +29,43 @@ export function UpdateHackerForm({
...
+          console.log("Form Errors:", form.formState.errors);
```
Comment (Lewin-B) [inline apps/admin/src/app/(dashboard)/hackers/update-hacker-form.tsx]: Do we want this?

**Review by Lewin-B — CHANGES_REQUESTED** [review]
small changes

File: apps/admin/src/app/(dashboard)/hackers/update-hacker-form.tsx (hunk 1 above)
Comment (DanyPalma) [reply apps/admin/src/app/(dashboard)/hackers/update-hacker-form.tsx]: removed

File: apps/admin/src/app/(dashboard)/hackers/update-hacker-form.tsx (hunk 2 above)
Comment (DanyPalma) [reply apps/admin/src/app/(dashboard)/hackers/update-hacker-form.tsx]: removed logs

**Review by Lewin-B — APPROVED** [review]
lgtm

**Pull request merged**